doc/styles/Vocab/ANSYS/accept.txt

```diff
@@ -128,18 +129,23 @@ PyAnsys
 PyAnsys Math
 PyDPF-Core
 PyDPF-Post
-PyMAPDL
 pymapdl
+PyMAPDL
 Python
 QRDAMP
 Radiosity
 Rao
 righ
 RNNN
+sbatch

```

Comment:
Weird that you'd have to add command names to this list as they should always be tagged code entities or in code blocks. Vale is supposed to be ignored these.

Reply:
Yeah... Vale... You know.